indico/htdocs/sass/partials/_inputs.scss

```diff
@@ -444,6 +444,19 @@ $dropdown-transition: $dropdown-transition-step ease-out;
                 width: 10em;
             }
 
+            > .form-row.i-has-action.i-synced-input {
```

Comment:
The reason for it is CSS selectivity, but looking at it, I might remove the `i-has-action` class from the selector.